src/__experimental__/components/input/input.stories.js

```diff
@@ -0,0 +1,215 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import {
+  boolean, number, select
+} from '@storybook/addon-knobs';
+import { Store, State } from '@sambego/storybook-state';
+import { dlsThemeSelector, classicThemeSelector } from '../../../../.storybook/theme-selectors';
+import OptionsHelper from '../../../utils/helpers/options-helper';
+import info from './documentation/info';
+import { Checkbox } from '../checkbox';
+import Decimal from '../decimal';
+import GroupedCharacter from '../grouped-character';
+import NumberInput from '../number';
+import DateInput from '../date';
+import { RadioButtonGroup, RadioButton } from '../radio-button';
+import { Select, Option } from '../select';
+import Switch from '../switch';
+import Textarea from '../textarea';
+
+const store = new Store({
+  numberInputValue: '',
+  groupedCharacterValue: '',
+  checkboxValue: false,
+  dateValue: '2009-12-12',
+  radioButtonGroupValue: ''
+});
+
+const getCommonProps = () => {
+  const percentageRange = {
+    range: true,
+    min: 0,
+    max: 100,
+    step: 1
+  };
+  const labelInline = boolean('labelInline', false);
+  const labelWidth = labelInline ? number('labelWidth', 30, percentageRange) : undefined;
+  const inputWidth = labelInline ? number('inputWidth', 70, percentageRange) : undefined;
+  const labelAlign = labelInline ? select('labelAlign', OptionsHelper.alignBinary) : undefined;
+
+  return {
+    labelInline,
+    labelWidth,
+    inputWidth,
+    labelAlign,
+    disabled: boolean('disabled', false),
+    readOnly: boolean('readOnly', false),
+    size: select('size', OptionsHelper.sizesRestricted, 'medium')
+  };
+};
+
+const InputIntegration = () => {
+  const commonProps = getCommonProps();
```

Comment:
"Knobs" would be a more specific name than "props".
```suggestion
  const commonKnobs = getCommonKnobs();
```

Reply:
good point, changed

Reply:
Thanks :white_check_mark: